tests/Common/BranchComponentTest.php

```diff
@@ -78,19 +83,22 @@ function ($branch) use ($branchName) {
             (new ListComponentConfigurationsOptions())->setComponentId($componentId)
         );
 
-        // creating new configuration in main branch shouldn't create new configuration in dev branch
-        $this->assertEmpty($configs);
+        // there should be the one config existing in production before creating the branch
+        $this->assertCount(1, $configs);
```

Comment:
```suggestion
        $this->assertCount(1, $branchConfigs);
```
sorry tu som to blbo pomenoval a asi by som to premenoval aby bolo jasne ze sa testuju branch configy

Reply:
přejmenováno - je tam toho přejmenování víc, jak se to kopíruje tak to začíná být zmatek... :D Protože máš "main-1" konfiguraci, která existuje v main ale i v branchi :D

Reply:
a jo :/